plugins/docker_swarm.yaml

```diff
@@ -0,0 +1,87 @@
+# Plugin Info
+version: 0.0.1
+title: Docker Swarm
+description: Log parser for Docker Swarm
+parameters:
+  message_log_path:
+    label: Message Logs Path
+    description: The absolute path to the Docker Swarm message logs
+    type: string
+    default: "/var/log/messages/*.log"
+  start_at:
+    label: Start At
+    description: Start reading file from 'beginning' or 'end'
+    type: enum
+    valid_values:
+      - beginning
+      - end
+    default: end
+
+# Set Defaults
+{{$message_log_path := default "/var/log/messages/*.log" .message_log_path}}
```

Comment:
Maybe out of scope for this, but this path looks very wrong. `/var/log/messages` is not specific to docker swarm, and also is unlikely to exist on most systems. It seems like we should be using the `journald` operator for a source.

Reply:
I can't definitively say. The default path was pulled directly from the docker swarm log bundle. This might be one that we'll have to test against an actual environment first.